### PR TITLE
Qf 3014 export qfieldcloud project from qfield

### PR DESCRIPTION
**Pull request by SeqLaz**
Added in export individual dataset section option to export files from a QFieldCloud project in QField, also added some improvement in the documentation

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3014 Extracting data synchronize from QFieldCloud using USB  cable in Android](https://app.clickup.com/t/2192114/QF-3014)

**Comment by m-kuhn** [comment]
@SeqLaz quality of life improvement: if you run the following commands in your local git repository, it will do the formatting for you:

```
pip install pre-commit
pre-commit install
```

**Comment by SeqLaz** [comment]
> @SeqLaz quality of life improvement: if you run the following commands in your local git repository, it will do the formatting for you:
> 
> ```shell
> pip install pre-commit
> pre-commit install
> ```

Great, thank you

File: documentation/get-started/storage.en.md [hunk 1]
```diff
@@ -126,6 +126,29 @@ Users can select the ‘Send to‘ action for individual datasets, allowing for
...
+4. Now, click on the three dots (⋮) located on the right side of the file or folder.
+
+!![](../assets/images/export-qfieldcloud-files-from-qfield-3-three-dots.png)
+
+5. Choose between the functions 'Send to...' or 'Export to folder...' based on your preference and follow the prompts accordingly."
+
+!![](../assets/images/export-qfieldcloud-files-from-qfield-4-options-to-send.png)
+
+!!! note
+    This functionality is only available on Android.
```
Comment (suricactus) [inline documentation/get-started/storage.en.md:150]:
```suggestion
    This functionality is available on Android only.
```

Just to make it coherent with the other PR. No idea which way is more prelevant in the docs already.

File: documentation/get-started/storage.en.md [hunk 2]
```diff
@@ -126,6 +126,29 @@ Users can select the ‘Send to‘ action for individual datasets, allowing for
 edited datasets directly to third party apps such as Gmail, Drive, Dropbox, Nextcloud,
 <insert your favourite messenger app>, etc.
 
+To export the layers from a synchronized QFieldCloud project, either on your device or a preferred cloud provider. To do this, within your project:
+
+1. Click on the 'Gear icon' located within your project.
+
+!![](../assets/images/export-qfieldcloud-files-from-qfield-1-gear-icon.png)
+
+2. Select 'Open Project Folder.'
```
Comment (suricactus) [inline documentation/get-started/storage.en.md:135]:
```suggestion
2. Select 'Open Project Folder' menu item.
```

File: documentation/get-started/storage.en.md [hunk 3]
```diff
@@ -126,6 +126,29 @@ Users can select the ‘Send to‘ action for individual datasets, allowing for
...
+
+!![](../assets/images/export-qfieldcloud-files-from-qfield-2-open-folder-project.png)
+
+3. Inside this project folder, you will find your layers stored in a GeoPackage named 'data.gpkg', also you can export you media attached (DCIM (Pictures), Audios Videos, etc).
```
Comment (suricactus) [inline documentation/get-started/storage.en.md:139]:
```suggestion
3. Inside this project folder, you will find your project files. Offline layers will be stored in a file named 'data.gpkg'. You can also export your attached files (Photos, Audio, Video, etc).
```

File: documentation/get-started/storage.en.md [hunk 4]
```diff
@@ -126,6 +126,29 @@ Users can select the ‘Send to‘ action for individual datasets, allowing for
...
+
+4. Now, click on the three dots (⋮) located on the right side of the file or folder.
+
+!![](../assets/images/export-qfieldcloud-files-from-qfield-3-three-dots.png)
+
+5. Choose between the functions 'Send to...' or 'Export to folder...' based on your preference and follow the prompts accordingly."
```
Comment (suricactus) [inline documentation/get-started/storage.en.md:145]:
```suggestion
5. Choose between the actions 'Send to...' or 'Export to folder...' based on your preference and follow the prompts accordingly.
```

**Review by suricactus — COMMENTED** [review]
Thanks, small English and formatting comments.

**Review by suricactus — APPROVED** [review]

**Pull request merged**